unified-codes/apps/web/src/components/containers/explorer/ExplorerPage.tsx

```diff
@@ -66,7 +66,7 @@ export const ExplorerPageComponent: ExplorerPage = ({
       classes={classes}
       table={<ExplorerTable />}
       toggleBar={<ExplorerToggleBar />}
-      searchBar={<ExplorerSearchBar />}
+      searchBar={<ExplorerSearchBar helperText={'Use * to indicate any characters'} />}
```

Comment:
As this is text, there's no need for `{'`
( I prefer the standard of double-quoting component parameters, though I see that the line below isn't, so you may prefer to go with consistency )

```suggestion
      searchBar={<ExplorerSearchBar helperText="Use * to indicate any characters"/>}
```

Reply:
Oh good point :) thanks